src/module.js

```diff
@@ -842,6 +842,7 @@ export default class DailyIframe extends EventEmitter {
       this._joinedCallback = (participants, error) => {
         if (this._meetingState === DAILY_STATE_ERROR) {
           reject(error);
+          return;
```

Comment:
while you're at this, mind adding the exact same fix where `this._loadedCallback` is defined here: https://github.com/daily-co/daily-js/blob/main/src/module.js#L761

Reply:
done!